lisa/_kmod.py

```diff
@@ -2267,4 +2285,23 @@ def from_target(cls, target, **kwargs):
     def _event_features(cls, events):
         return set(f'event__{event}' for event in events)
 
+    def install(self, kmod_params=None):
+        try:
+            self.uninstall()
+        except Exception:
+            pass
+
+        probe_cmd = 'modprobe {} {} version={}'.format(
+            f"-d /vendor_dlkm/lib/modules/{self.target.kernel_version.release}" if self.target.os == 'android' else "",
+            quote(self.src.mod_name), self.src.checksum_sources
+        )
+
+        try:
+            self.target.execute(probe_cmd)
+            self.logger.info(f'Loaded module {self.src.mod_name} from kernel modules directory')
```

Comment:
nit: We are not trying to catch issues in `self.logger.info()`, it should therefore not be in the `try` clause. Instead it should be in the `else` clause.